.circleci/config.yml

```diff
@@ -4,69 +4,72 @@ aliases:
   - &working_directory ~/merchant-center-application-kit
 
   - &restore_yarn_cache
-    name: 'Restoring yarn cache'
-    keys:
-      - v1-yarn-cache-{{ checksum "yarn.lock" }}
-      - v1-yarn-cache
+    restore_cache:
+      name: 'Restoring yarn cache'
+      keys:
+        - v1-yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+        - v1-yarn-cache-{{ .Branch }}
+        - v1-yarn-cache
 
   - &save_yarn_cache
-    name: 'Saving yarn cache'
-    key: v1-yarn-cache-{{ checksum "yarn.lock" }}
-    paths:
-      - ~/.cache/yarn
-      - node_modules
+    save_cache:
+      name: 'Saving yarn cache'
+      key: v1-yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      paths:
+        - ~/.cache/yarn
+
+  - &run_yarn
+    run: yarn --frozen-lockfile
```

Comment:
It's recommended to use `--frozen-lockfile` on CI

Reply:
I think we did that before or at laest elsewhere.